src/Console/ConsoleConfig.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Ushahidi\Console;
+
+use Aura\Di\Container;
+use Aura\Di\ContainerConfig;
+
+class ConsoleConfig extends ContainerConfig
+{
+    public function define(Container $di)
+    {
+        $di->set('authorizer.console', $di->lazyNew('Ushahidi\Console\Authorizer\ConsoleAuthorizer'));
+    }
+
+    public function modify(Container $di)
+    {
+    }
+}
```

Comment:
These are very similar to ServiceProviders for Laravel. 
define ~= register
modify ~= boot